### PR TITLE
Adjust for latest roxygen2 update

### PR DESCRIPTION
**Pull request by pat-s**
https://github.com/klutometis/roxygen/issues/822

### CODE REVIEW

File: tic.R [hunk 1]
```diff
@@ -9,6 +9,6 @@ if (Sys.getenv("id_rsa") != "") {
     add_step(step_setup_ssh())
 
   get_stage("deploy") %>%
-    add_step(step_build_pkgdown()) %>%
+    add_step(step_build_pkgdown(document = FALSE)) %>%
```
Comment (krlmlr) [inline tic.R]: This seems to make sense as a default setting for _tic_ too.

**Comment by edzer** [comment]
@pat-s any idea what is going on [here](https://travis-ci.org/r-spatial/lwgeom/jobs/490621986)?

**Comment by edzer** [comment]
@pat-s any idea what is going on here: https://travis-ci.org/r-spatial/lwgeom/jobs/516967717 ?

**Comment by krlmlr** [comment]
Looks like a {pkgdown} error to me. Can you build the pkgdown site locally?

**Comment by edzer** [comment]
Indeed! It stops at
```r
> pkgdown::build_home(quiet = FALSE)
── Building home ───────────────────────────────────────────────────────────────
Error in xml2::xml_contents(heading)[[1]] : subscript out of bounds
> traceback()
5: xml2::xml_add_sibling(xml2::xml_contents(heading)[[1]], "a", 
       href = paste0("#", anchor[[i]]), class = "anchor", .where = "before")
4: tweak_anchors(xml, only_contents = FALSE)
3: markdown(src_path)
2: build_home_index(pkg, quiet = quiet)
1: pkgdown::build_home(quiet = FALSE)
```
but there I get lost. Any suggestion?

**Comment by pat-s** [comment]
The error appeared after https://github.com/r-spatial/lwgeom/commit/f0167391659e941ebe5c7352e2443b2cc6c5dafa which introduced the roxygen2 v6.1.1 Note.

`RoxygenNote: 6.1.1` should not be a problem actually and pkgdown v1.3.0 is also fine.

The log complains about the authors field. Maybe caused by `person("liblwgeom developers", role = "cph"))`? However, this did not change recently and pkgdown built fine before.

I'll try some changes, maybe we need again to chain `devtools::document()` as we do in other packages since roxygen2 >= 6.1.1.

**Comment by pat-s** [comment]
Ok, found the issue.

After a fresh clone of the repo, README.md looks like this:

screenshot_2019-04-08_21-30-20_348594184

Note the missing title in the last Header2. This causes the error shown above.
The README on GH looks fine so something seems to corrupt the cloning process.
Since Travis also just clones the repo, the error is also occurring there.

**Comment by edzer** [comment]
Thank you so much!

**Comment by pat-s** [comment]
You're welcome.
Any idea about the GH <-> local mismatch?

**Comment by edzer** [comment]
I guess GH is more foregiving about mistakes like this.

**Pull request merged**